src/app/features/ledger/flows/jwt-signing/ledger-sign-jwt-container.tsx

```diff
@@ -91,7 +91,11 @@ export function LedgerSignJwtContainer() {
 
     const stacks = await prepareLedgerDeviceStacksAppConnection({
       setLoadingState: setAwaitingDeviceConnection,
-      onError() {
+      onError(e) {
+        if (e instanceof Error && checkLockedDeviceError(e)) {
+          setLatestDeviceResponse({ deviceLocked: true } as any);
```

Comment:
Maybe we can add a specific type for this to avoid using `any`? 

We should probably have a lint rule banning `any`

Reply:
Good point! I'll open a separate issue for this one as it seems like we need to refactor latestDeviceResponse state for this one, and i'm a bit hesitant to touch it right now as it might break smth in weird ways.